workers/default/image/image_scaling.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os
+
+from PIL import Image as PilImage
+
+
+def scale_image(image_path, max_width, max_height, target_dir):
+    """
+    Scale the image to the given size.
+
+    Open the given image and scales the size to the given values for
+    width and height.
```

Comment:
And keeps the side-ratio. Wie du schon richtig benennst sind das ja max_height und width, das Seitenverhältnis bleibt also erhalten. Wäre mMn wichtig da im Docstring drauf hinzuweisen.